packages/config/src/projects/other/da-beat/blockchain/celestia/bridges/blobstream/SP1Blobstream/ethereum.ts

```diff
@@ -0,0 +1,99 @@
+import { EthereumAddress } from '@l2beat/shared-pure'
+import { ProjectDiscovery } from '../../../../../../../../discovery/ProjectDiscovery'
+import { DaAttestationSecurityRisk } from '../../../../../types/DaAttestationSecurityRisk'
+import { DaExitWindowRisk } from '../../../../../types/DaExitWindowRisk'
+import { CELESTIA_BLOBSTREAM } from '../template'
+
+const discovery = new ProjectDiscovery('blobstream')
+
+const maxRangeDataCommitment = discovery.getContractValue<number>(
+  'Blobstream',
+  'DATA_COMMITMENT_MAX',
+)
+
+const relayers = discovery.getContractValue<string[]>('Blobstream', 'relayers')
+
+const blobstreamVerifier = discovery.getContractValue<string>(
+  'SuccinctGatewaySP1',
+  'blobstreamVerifier',
+)[0]
+
+export const blobstreamEthereum = CELESTIA_BLOBSTREAM({
+  chain: 'ethereum',
+  usedIn: [
+    // no project integrates it for state validation
+  ],
+  display: {
+    links: {
+      websites: [],
+      documentation: ['https://docs.celestia.org/developers/blobstream'],
+      repositories: ['https://github.com/succinctlabs/sp1-blobstream'],
+      apps: [],
+      explorers: ['https://etherscan.io/'],
+      socialMedia: [],
+    },
+  },
+  technology: `
+     The Blobstream bridge is a data availability bridge that facilitates data availability commitments to be bridged between Celestia and Ethereum.
+     The Blobstream bridge is composed of three main components: the **Blobstream** contract, the **Succinct Gateway** contracts and the **Verifier** contracts.  <br /> 
```

Comment:
'between Celestia and Ethereum' should be 'between Celestia and the targetchain'
If the text is 100% the same in all three deployments you could also save it somewhere else and only change this variable targetchain

Reply:
Thanks! added variables, we can't fetch chain name and update interval by chain anywhere, but they still help